rick_and_morty_kobe/pubspec.yaml

```
@@ -0,0 +1,92 @@
+# The following section is specific to Flutter packages.
+flutter:
+
+  # The following line ensures that the Material Icons font is
+  # included with your application, so that you can use the icons in
+  # the material Icons class.
+  uses-material-design: true
+
+  # To add assets to your application, add an assets section, like this:
+  assets:
+    - assets/images/
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)